test_driver.py

```diff
@@ -102,45 +81,52 @@ def LoadManifest(manifest_filename, manifest_args):
     sys.exit(3)
 
 
-def ParseStl(stl_file, modules, parse_env):
-  m = NewModuleDict()
-  stl.parser.Parse(stl_file, m, parse_env)
-  # TODO(byungchul): Support module in multiple files.
-  modules[m['name']] = m
+def ParseStl(stl_file, global_env):
+  stl.parser.Parse(stl_file, global_env)
 
 
-def LoadModules(manifest, test_manifest_filename, parse_env):
+def LoadModules(manifest, test_manifest_filename, global_env):
   """Builds transition graph for each module."""
-  modules = {}
+  global_env = {'modules': {}}
   if 'stl_files' in manifest:
     for f in manifest['stl_files']:
       f = os.path.join(os.path.dirname(test_manifest_filename), f)
-      ParseStl(f, modules, parse_env)
-  logging.debug(str(modules))
-  return modules
+      ParseStl(f, global_env)
+  logging.debug(str(global_env['modules']))
+  return global_env['modules']
 
 
 def FillInModuleRoles(modules, manifest):
   """Fills in role information in |modules|."""
   for r in manifest['roles']:
     module, name = r['role'].split('::', 1)
-    if name not in modules[module]['roles']:
+    if name not in modules[module].roles:
       raise NameError("Cannot find a role in module '%s': %s" % (module, name))
-    role = modules[module]['roles'][name]
+    role = modules[module].roles[name]
     for v in r:
       if v == 'role':
         continue
       role[v] = r[v]
 
 
+def FillInConstants(modules, manifest):
+  """Fills in constant information in |modules|."""
+  for key, val in manifest['constants'].iteritems():
+    module, name = key.split('::', 1)
+    if name not in modules[module].consts:
+      raise NameError("Cannot find a constant in module '%s': %s" % (module,
+                                                                     name))
+    modules[module].consts[name].value = val
```

Comment:
Please check if value is already set. Print warning if we want to allow overriding, otherwise, raise Error.